### PR TITLE
[PR] Update project structure, query for all labs sites

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "washingtonstateuniversity/wsuwp-spine-child-theme-skeleton",
```
Comment (philcable) [inline composer.json]: One more `skeleton` reference :)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,366 @@
+WSUWP Spine Child Theme Skeleton
```
Comment (philcable) [inline LICENSE]: This should be updated to `WSU Labs` (or whatever is most appropriate).

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+consider it more useful to permit linking proprietary applications with the
+library.  If this is what you want to do, use the GNU Lesser General
+Public License instead of this License.
+
+WRITTEN OFFER
+
+The source code for any program binaries or compressed scripts that are
+included with WSU Content Visibility can be freely obtained at the following URL:
+
+	https://github.com/washingtonstateuniversity/WSUWP-Spine-Child-Theme-Skeleton/
```
Comment (philcable) [inline LICENSE]: This `skeleton` reference and the `WSU Content Visibility` one on line 364 should be updated, too.

**Review by philcable — APPROVED** [review]
👍  Looks good! I left a few inline comments about references from the skeleton structure that should be updated, otherwise good to go!

**Pull request merged**